isofit/utils/apply_oe.py

```diff
@@ -379,6 +379,11 @@ def apply_oe(args):
         paths.loc_working_path, lut_params, pressure_elevation=args.pressure_elevation
     )
 
+    if lut_params.flag_ocean_elevation:
+        elevation_lut_grid = None
+        # mean_elevation_km = 0.0001 #KF emulator
+        mean_elevation_km = 0.0  # MODTRAN
+
```

Comment:
is this needed?  Seems we could handle this with the lut_config parameter to avoid introducing new fields.

Reply:
Agreed. And moreover, the code already handles cases of both negative elevation and `len(elevation_grid) == 1`. @reginaeckert Could you briefly clarify what you had in mind here?

Reply:
This was a stopgap to avoid the LUT including many elevations when land happened to be in the scene. Setting the spacing directly makes sense to me.

Reply:
Removed with commit [465edd3](https://github.com/isofit/isofit/pull/521/commits/465edd3a121f18cda57c4bb31f8876976dd1960d).